test/postman/dbp-test-collection.json

```diff
@@ -1677,6 +1161,54 @@
 					}
 				}
 			]
+		},
+		{
+			"name": "06 Activiti-token-auth",
+			"item": [
+				{
+					"name": "getProcessInstances",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"id": "1fcaf9d8-243d-44c5-92f1-681cfaedba35",
+								"exec": [
+									"pm.test(\"Response code is 200\", function () {",
+									"    pm.response.to.have.status(200);",
+									"});"
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "GET",
+						"header": [
+							{
+								"key": "Authorization",
+								"value": "Bearer {{kcAccessToken}}"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": ""
+						},
+						"url": {
+							"raw": "https://{{activitiUrl}}/rb-my-app/v1/process-instances",
+							"protocol": "https",
+							"host": [
+								"{{activitiUrl}}"
+							],
+							"path": [
+								"rb-my-app",
+								"v1",
+								"process-instances"
+							]
+						}
+					},
+					"response": []
+				}
+			]
```

Comment:
At this point we have established that we can log in using the Identity Service token. It may be worth checking whether we can actually get some information too (I don't know how the API for this looks like in APS2).

Reply:
I don't understand what kind of information are you expecting to be checked by tests.

Reply:
Whether the user can actually do something once logged on. For APS1.9, if I'm not mistaken, we were checking whether this user could see some predefined process definition (though I'm not sure if the same principle can be applied to APS2.x).

Reply:
As far as activiti is concerned, we only test read operations. In the APS 1.9 tests we wanted to make sure that the test user could start and list tasks etc. These tests have been removed and not replaed with anything APS 2.1 specific and in my opinion this is insufficient to give us sufficient confidence about DBP deployment.
Please refer to the Task Controller API documentation for implementation details: http://activiti-cloud-gateway.52.16.22.192.nip.io/rb-my-app/swagger-ui.html#/task-controller-impl